src/Workspaces/Core/Portable/FindSymbols/SymbolTree/SymbolTreeInfo_Source.cs

```diff
@@ -54,7 +54,7 @@ private static void FreeSymbolMap(MultiDictionary<string, ISymbol> symbolMap)
 
             // Order the documents by FilePath.  Default ordering in the RemoteWorkspace is
```

Comment:
That seems strange though... the way your system syncs, i though you would be sending from VS to OOP the projects in order, and the documents in order in those projects.  These checksums would be how you would sync everything over, and you'd use them to order things on the OOP side appropriately.